src/test/components/elements/panels/SelectedEventDetails/countGenders.test.tsx

```diff
@@ -0,0 +1,20 @@
+import { EventBookingDTO } from "../../../../../IsaacApiTypes";
+import { countGenders } from "../../../../../app/components/elements/panels/SelectedEventDetails";
+import { mockCancelledEventBooking, mockEventBookings } from "../../../../../mocks/data";
+
+describe("countGenders", () => {
+  it("returns an object with all genders set to 0 when passed an empty array", () => {
+    const result = countGenders([]);
+    expect(result).toEqual({ male: 0, female: 0, other: 0, preferNotToSay: 0, unknown: 0 });
+  });
+
+  it("should return the correct count of genders when passed an array of Event Bookings", () => {
```

Comment:
This test and the next one are not easy to understand because every event booking has a million properties and for these tests you only care about two: the gender and the booking status. But that's not clear by reading this test, I only know that because I read the implementation, which is a sign that the test is not great. Instead, what you can do, is to load the mocks and explicitly set the gender and booking status in this test (and the next one). That way it's more explicit and clear, the test becomes very easy to understand.